quinn-proto/src/transport_parameters.rs

```diff
@@ -160,6 +166,17 @@ impl TransportParameters {
             min_ack_delay: Some(
                 VarInt::from_u64(u64::try_from(TIMER_GRANULARITY.as_micros()).unwrap()).unwrap(),
             ),
+
+            receive_timestamps_exponent: config
+                .ack_timestamp_config
+                .as_ref()
+                .map(|cfg| cfg.exponent),
+
+            max_recv_timestamps_per_ack: config
+                .ack_timestamp_config
+                .as_ref()
+                .map(|cfg| cfg.max_timestamps_per_ack),
+
```

Comment:
I think these maybe shouldn't be separate `Option`s, but a tuple wrapped in a single `Option`?

Reply:
> It could be worth proposing for the draft that the transport parameters related to this feature are grouped together similar to how PreferredAddress does it.

Yes, that would probably be good.